docs/source/readout_noise.rst

```diff
@@ -0,0 +1,354 @@
+
+.. math::
+
+
+   \rho_{\text{post measurement}} = \sum_{x\in\mathcal{O}} p(x) \rho_x = \sum_{x\in\mathcal{O}} \Pi_x \rho \Pi_x.
+
+The completeness of the projector set ensures that the trace of the
+post measurement is still 1 and the Kraus map form of this expression
+ensures that :math:`\rho_{\text{post measurement}}` is a positive
+(semi-)definite operator.
+
+Classical readout bit-flip error
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider now the ideal measurement as above, but where the outcome
+:math:`x` is transmitted across a noisy classical channel that produces
+a final outcome :math:`x'\in \mathcal{O}' = \{0', 1'\}` according to
+some conditional probabilities :math:`p(x'|x)` that can be recorded in
+the *assignment probability matrix*
+
+.. math::
+
+
+   P_{x'|x} = \begin{pmatrix}
+   p(0 | 0) & p(0 | 1) \\
+   p(1 | 0) & p(1 | 1)
+   \end{pmatrix}
+
+Note that this matrix has only two independent parameters as each
+column must be a valid probability distribution, i.e. all elements are
+non-negative and each column sums to 1.
+
+This matrix allows us to obtain the probabilities
+:math:`\mathbf{p}' := (p(x'=0), p(x'=1))^T` from the original outcome
+probabilities :math:`\mathbf{p} := (p(x=0), p(x=1))^T` via
+:math:`\mathbf{p}' = P_{x'|x}\mathbf{p}`. The difference relative to the
+ideal case above is that now an outcome :math:`x' = 0` does not
+necessarily imply that the post measurement state is truly
+:math:`\Pi_{0} \rho \Pi_{0} / p(x=0)`. Instead, the post measurement
+state given a noisy outcome :math:`x'` must be
+
+.. raw:: latex
+
+   \begin{align}
+   \rho_{x'} & = \sum_{x\in \mathcal{O}} p(x|x') \rho_x \\
+             & = \sum_{x\in \mathcal{O}} p(x'|x)\frac{p(x)}{p(x')} \rho_x \\
+             & = \frac{1}{p(x')}\sum_{x\in \mathcal{O}} p(x'|x) \Pi_x \rho \Pi_x
+   \end{align}
+
+where
+
+.. raw:: latex
+
+   \begin{align}
+   p(x') & = \sum_{x\in\mathcal{O}} p(x'|x) p(x)  \\
+   & = \tr{\sum_{x\in \mathcal{O}} p(x'|x) \Pi_x \rho \Pi_x} \\
+   & = \tr{\rho \sum_{x\in \mathcal{O}} p(x'|x)\Pi_x} \\
+   & = \tr{\rho E_x'}.
+   \end{align}
+
+where we have exploited the cyclical property of the trace
+:math:`\tr{ABC}=\tr{CBA}` and the projection property
+:math:`\Pi_x^2 = \Pi_x`. This has allowed us to derive the noisy outcome
+probabilities from a set of positive operators
+
+.. math::
+
+
+   E_{x'} := \sum_{x\in \mathcal{O}} p(x'|x)\Pi_x \ge 0
+
+that must sum to 1:
+
+.. math::
+
+
+   \sum_{x'\in\mathcal{O}'} E_x' = \sum_{x\in\mathcal{O}}\underbrace{\left[\sum_{x'\in\mathcal{O}'} p(x'|x)\right]}_{1}\Pi_x = \sum_{x\in\mathcal{O}}\Pi_x = 1.
+
+The above result is a type of generalized **Bayes' theorem** that is
+extremely useful for this type of (slightly) generalized measurement and
+the family of operators :math:`\{E_{x'}| x' \in \mathcal{O}'\}` whose
+expectations give the probabilities is called a **positive operator
+valued measure** (POVM). These operators are not generally orthogonal
```

Comment:
It took me a while to parse this sentence starting with "The above result ..." maybe refactor into two sentences? :)